src/main/java/org/commcare/formplayer/services/ScheduledTasks.java

```diff
@@ -32,6 +32,9 @@ public class ScheduledTasks {
     @Autowired
     private VirtualDataInstanceService virtualDataInstanceService;
 
+    @Autowired
+    private MediaHandler mediaHandler;
```

Comment:
I don't think this will work as MediaHandler requires a `MultipartFile` attribute to be initialised which we are not supplying anywhere. I think the best approach here is to move `purge` method in `MediaMetaDataService` instead and then call `cleanMedia` from there. 

````
val deletedSuccessfully = MediaHandler.cleanMedia(parentPath, fileIdWithExt)
````

To do this we will also need to make `cleanMedia` static so that we don't need to initialise `MediaHandler` separaterly for this call. I am going to make some small refactors on this branch to be able to do that.

Reply:
>MediaHandler requires a MultipartFile attribute to be initialised which we are not supplying anywhere

ah, I missed that outside of the test.

Reply:
Ok I made a change  [here](https://github.com/dimagi/formplayer/pull/1316/commits/03930c4f38485307e2598c4e1fdba1eb2674290f). Transitioning to you to make the change to move `purge` to `MediaMetaDataService` instead.

Reply:
I'm running into a weird blocker where "`this` is not available" in the debugger. That is making the test fail and making it hard to reason about what is and isn't working in moving the `purge` method to the service. Have you run into this before?